pkg/controller/controller_test.go

```diff
@@ -846,7 +848,55 @@ func TestProvision(t *testing.T) {
 				},
 			},
 		},
-		"provision with access modes": {
+		"provision with access mode multi node multi writer": {
```

Comment:
Add a test with AccessModes with multiple mode specified (i.e. {ReadWriteOnce, ReadOnlyMany})

Reply:
You got me on that one! :-D

Reply:
Basically add test cases with multiple modes.